dev/TOC.md

```diff
@@ -350,12 +328,47 @@
     - [PD](reference/key-monitoring-metrics/pd-dashboard.md)
     - [TiKV](reference/key-monitoring-metrics/tikv-dashboard.md)
   - [Adopters](adopters.md)
++ TiDB in Kubernetes
+  - [About TiDB Operator](tidb-in-kubernetes/overview.md)
+  + Get Started
+    - [DinD](tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-dind.md)
+    - [GKE](tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-gke.md)
+    - [Minikube](tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-minikube.md)
+  + Deploy
+    - [Prerequisites](tidb-in-kubernetes/deploy/prerequisites.md)
+    - [TiDB Operator](tidb-in-kubernetes/deploy/tidb-operator.md)
+    - [TiDB in General Kubernetes](tidb-in-kubernetes/deploy/general-kubernetes.md)
+    - [TiDB in AWS EKS](tidb-in-kubernetes/deploy/aws-eks.md)
+    - [TiDB in GCP](how-to/tidb-in-kubernetes/gcp-gke.md)
+    - [TiDB in Alibaba Cloud](how-to/tidb-in-kubernetes/alibaba-cloud.md)
+  + Configure
+    - [Cluster Initialization](tidb-in-kubernetes/initialize-cluster.md)
+  - [Monitor](tidb-in-kubernetes/monitor/tidb-in-kubernetes.md)
+  + Maintain
+    - [Destroy a TiDB cluster](tidb-in-kubernetes/maintain/delete-cluster.md)
+    - [Maintain a Hosting Kubernetes Node](tidb-in-kubernetes/maintain/kubernetes-node.md)
+    - [Backup and Restore](tidb-in-kubernetes/maintain/backup-and-restore.md)
+    - [Collect Logs](tidb-in-kubernetes/maintain/log-collecting.md)
+    - [Automatic Failover](tidb-in-kubernetes/maintain/auto-failover.md)
+  - [Scale](tidb-in-kubernetes/scale-in-kubernetes.md)
+  + Upgrade
+    - [TiDB Cluster](tidb-in-kubernetes/upgrade/tidb-cluster.md)
+    - [TiDB Operator](tidb-in-kubernetes/upgrade/tidb-operator.md)
+  + Reference
+    + Configuration
+      - [TiDB Cluster](tidb-in-kubernetes/reference/configuration/tidb-cluster.md)
+      - [Backup](tidb-in-kubernetes/reference/configuration/backup.md)
+      - [Local PV](tidb-in-kubernetes/reference/configuration/local-pv.md)
+    + Tools
+      - [tkctl](tidb-in-kubernetes/reference/tools/tkctl.md)
+      - [Tools in Kubernetes](tidb-in-kubernetes/reference/tools/in-kubernetes.md)
```

Comment:
Where is this `Tools in Kubernetes` document?

Reply:
Not merged yet. Tom is reviewing it.

Reply:
Ok.

Reply:
Review finished.